src/plugins/jqplot.pointLabels.js

```diff
@@ -273,7 +273,7 @@
         for (var i=0; i<p._elems.length; i++) {
             // Memory Leaks patch
             // p._elems[i].remove();
-            p._elems[i].emptyForce();
+            if(p._elems[i]) p._elems[i].emptyForce();
```

Comment:
Good catch! Thanks.
But I would like to see the if statement accompanied with curly braces.
```
 if (p._elems[i]) {
      p._elems[i].emptyForce();
}
```
Can you adapt this please?
I understand we need to make code style guidelines more clear and visible. Your input is appreciated!